src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1.405-alpine3.12 AS build
+FROM mcr.microsoft.com/dotnet/sdk:5.0.102-1-alpine3.12 AS build
 WORKDIR /Authentication
 
 COPY Authentication .
 
 RUN dotnet build Altinn.Platform.Authentication.csproj -c Release -o /app_output
 RUN dotnet publish Altinn.Platform.Authentication.csproj -c Release -o /app_output
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.11-alpine3.12 AS final
+FROM mcr.microsoft.com/dotnet/aspnet:5.0.2-alpine3.12 AS final
```

Comment:
@TheTechArch Latest is `5.0.3-alpine3.13` and there are [security fixes](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0.3/5.0.3.md#notable-changes).

Reply:
Let's do that as a separate PR.